install.hs

```diff
@@ -247,7 +247,7 @@ installCabal = do
   -- install `cabal-install` if not already installed
   when (isNothing cabalExe) $
     execStackShake_ ["install", "cabal-install"]
-  execCabal_ ["update"]
+  execCabal_ ["v1-update"]
```

Comment:
I think we had this before, and it broke installs for people using earlier versions of cabal.

Reply:
Probably true, but we demand that the cabal version is at least `2.4.1.0`. this is checked before that.